frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/RestrictiveLegalValues/RestrictiveLegalValues.tsx

```diff
@@ -47,7 +47,9 @@ export const getIllegalValues = (
 ) => {
     const deletedValuesSet = getLegalValueSet(deletedLegalValues);
 
-    return constraintValues.filter((value) => deletedValuesSet.has(value));
+    return constraintValues.filter(
+        (value) => value !== '' && deletedValuesSet.has(value),
```

Comment:
This can probably be simplified just to this, right? 

```suggestion
        (value) => value && deletedValuesSet.has(value),
```

But also ... why would the deleted values set have `''` as a value? 🤷🏼 Eh.